cloud_utils/cache/stores/redis_sync.py

```diff
@@ -35,21 +36,24 @@ def get_item(key: str):
             logging.debug(f"{key} is not in {name}")
             raise KeyError
         try:
```

Comment:
I think this should be an explicit parameter and not use a heuristic to determine if you want to dumps and loads or not. 
Another option, maybe just use the decode/enode functions now that we have them ? wdyt ?

Reply:
Sounds good! 
I tried it at the beginning when writing with the decode and encode function and had some issues with it so I added an explicit parameter to the function now